testing/run_tests.py

```diff
@@ -95,7 +96,7 @@ def RunCCTests(build_dir, filter):
     ]
   RunEngineExecutable(build_dir, 'flow_unittests', filter, flow_flags + shuffle_flags)
 
-  RunEngineExecutable(build_dir, 'fml_unittests', filter, [ time_sensitve_test_flag ] + shuffle_flags)
+  RunEngineExecutable(build_dir, 'fml_unittests', filter, [ time_sensitve_test_flag, gpu_thread_merger_test_flag ] + shuffle_flags)
```

Comment:
Let's add a TODO and reference an issue to fix the flake? should probably be assigned to @iskakaushik

Reply:
done: https://github.com/flutter/flutter/issues/44002